### ISSUE TITLE
View differentiation

### ISSUE THREAD

**Issue by Prizephitah** [psequel/psequel#76]
Now that views are shown in the table list (#23, awsome!), it would be nice witch some indicator of which is which. As each table has a small icon to the left of the name, could w get a different icon for views?

**Comment by psequel** [comment]
PSequel is using different icons for tables and views already. Is it not the case for you?

**Comment by Prizephitah** [comment]
No, they all look like a 3x3 grid. Tested on two different machines, both running PSequel 1.3.4 and El Capitan.

**Comment by psequel** [comment]
Hmm...this is what I have in PSequel 1.3.4 and El Capitan. "items" is a table, "new_items" is a view.

![image](https://cloud.githubusercontent.com/assets/10246639/11055967/66c2ae1a-8731-11e5-8fbf-e8c7d8d728e3.png)

**Comment by Prizephitah** [comment]
Hmm, my problem seems to be that I can't see views at all. (I was fooled by tables named *_view). I even tried reinstalling. The database is running Postgres 9.3.

**Comment by psequel** [comment]
PSequel uses this query to fetch tables and views. Please let me know its output.
```
SELECT relname, nspname, relkind FROM pg_catalog.pg_class c, pg_catalog.pg_namespace n WHERE relkind IN ('r', 'v') AND nspname LIKE 'public' AND relname LIKE '%' AND nspname NOT IN ('pg_catalog', 'information_schema', 'pg_toast', 'pg_temp_1') AND n.oid = relnamespace ORDER BY nspname, relname
```

**Comment by Prizephitah** [comment]
Found the problem! I am looking for materialized views which has the relkind _m_. The following query gave all my tables and view. Note that I skipped the `nspname LIKE 'public'` condition as I work in many different schemas.

```SELECT relname, nspname, relkind FROM pg_catalog.pg_class c, pg_catalog.pg_namespace n WHERE relkind IN ('r', 'v', 'm') AND relname LIKE '%' AND nspname NOT IN ('pg_catalog', 'information_schema', 'pg_toast', 'pg_temp_1') AND n.oid = relnamespace ORDER BY nspname, relname```

**Comment by psequel** [comment]
Ahh I see. Currently PSequel doesn't include materialized views in the table list.

**Comment by bwiggs** [comment]
+1 for materialized view support.

**Comment by bsb20** [comment]
Same, this necessary to me for this to be useable.

**Comment by mshandrovskiy** [comment]
+1
